sessions/css-layout-with-flexbox/personal-travel-diary/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML & CSS</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+	</head>
+	<body>
+		<h1>My Personal Travel Diary</h1>
+		<main>
+			<section class="about">
+				<img
+					class="about-img"
```

Comment:
How nitpicky do we want to be about BEM standards? 🤔

Reply:
I change the code to use BEM in this exercise